AIO-Tools/packages.config

```diff
@@ -1,12 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Little question: why is that got deleted?

Reply:
packages.config is deprecated